apps/mobile/maestro/flows/hide-account.yaml

```diff
@@ -14,7 +14,7 @@ appId: io.leather.mobilewallet
     id: 'walletListAccountCard'
 - tapOn:
     text: 'Hide account'
-- assertVisible: 'Unhide account'
+- assertVisible: 'Un-hide account'
```

Comment:
I updated this test text as it was failing due to copy change from `Un-hide account` to `Unhide account`. 

This is a bit of a manual step but I think it's worth it as:
- the tests are easier to read
- it exposed that we aren't running these tests in CI as this should have been failing and blocking merges